tutor/utils.py

```diff
@@ -366,3 +366,130 @@ def format_table(rows: List[Tuple[str, ...]], separator: str = "\t") -> str:
             # Append EOL at all lines but the last one
             formatted += "\n"
     return formatted
+
+
+def get_mysql_change_charset_query(
```

Comment:
Let's move this function outside of tutor/utils.py to a more specific module. Maybe jobs_utils.py ?

Reply:
Would you prefer we keep jobs_utils.py at tutor/jobs_utils.py or at tutor/commands/jobs_utils.py?

Reply:
tutor/commands/jobs_utils.py would be more appropriate.